src/managing-workflow/upgrading-workflow.md

```diff
@@ -5,9 +5,9 @@ To upgrade to a newer version of Workflow, run the following:
 ```
 $ helmc uninstall workflow-beta3 -n deis
```

Comment:
We should update this line to "workflow-beta4" so it illustrates an upgrade from `workflow-beta4` to `workflow-rc1`.